openslides/agenda/main_menu.py

```diff
@@ -10,5 +10,5 @@ class AgendaMainMenuEntry(MainMenuEntry):
     verbose_name = ugettext_lazy('Agenda')
     required_permission = 'agenda.can_see'
     default_weight = 20
-    pattern_name = 'item_overview'
+    pattern_name = '/agenda'
```

Comment:
@normanjaeckel I used `pattern_name` only temporary (that we can use links on main menu entrys for testing) until we have fixed generic main menu with angular (#1469). I added TODOs in the main_menu.py code.